darasimiolaifa/covid-19-estimator-darasimiolaifa
# feat(challenge1): Calculate infected cases and make projections

## What does this PR do?
This PR fulfills the solution to the challenge 1 of the BuildForSDG assessment test

## What are the major activities carried out?
- create the calculateInfectedcases function
- create the calculateProjectedInfectedcases function
- refactor the functions into a separate file and export a helper function for accessing them

## How can this PR be manually tested?
- Clone the repo
- Install the dependencies
- Create a test file that imports the `covid19ImpactEstimator` function from the `src/estimator` file.
- Call the function with the example data like the one below:

```javascript

{
   region: {
     name: "Africa",
     avgAge: 19.7,
     avgDailyIncomeInUSD: 5,
     avgDailyIncomePopulation: 0.71
   },
   periodType: "days",
   timeToElapse: 58,
   reportedCases: 674,
   population: 66622705,
   totalHospitalBeds: 1380614
}
```
- You should get a response object that includes your input data in its `data` property, and two other properties namely `impact` and `severeImpact`
